workers/unity/Packages/io.improbable.gdk.buildsystem/Util/CompilerValidator.cs

```diff
@@ -0,0 +1,29 @@
+using System.Linq;
+using UnityEditor;
+using UnityEditor.Compilation;
+
+namespace Packages.io.improbable.gdk.buildsystem.Util
```

Comment:
This namespace should be `Improbable.Gdk.BuildSystem.Util`. Rider/VS will typically generate it based off the file path (which is why it looks as it does).